src/middle/Index.ml

```diff
@@ -43,6 +43,13 @@ let apply ~default ~merge op (ind : 'a t) =
   | Between (expr_top, expr_bottom) -> merge (op expr_top) (op expr_bottom)
   | MultiIndex exprs -> op exprs
 
+let map_expr ~f = function
```

Comment:
We already derive `map` for the type `t`, so this is equivalent to `Index.map`

Reply:
Yeah this needs deleted